File: Soumadip/string/CompareString.java
package Soumadip.string;

import java.util.*;
public class CompareString {
      
      public static void compare(String str1,String str2)
      {
            if(str1.equals(str2))
            {
                  System.out.println("Both input string are equal !!");
            }
            else{
                  System.out.println("Both input string are not equal !!");
            }
      }
      public static void compareLexicograph(String str1,String str2)
      {
            if(str1.compareTo(str2) == 0){
                  System.out.println("Both input string are equal !!");
            }
            else if(str1.compareTo(str2) > 0){
                  System.out.println("'"+str1+"' is lexicographically bigger than string '"+str2+"'");
            }
            else{
                  System.out.println("'"+str2+"' is lexicographically bigger than string '"+str1+"'");
            }
      }
      public static void compareIgnoreCase(String str1,String str2)
      {
            if(str1.compareToIgnoreCase(str2) == 0){
                  System.out.println("Both input string are equal !!");
            }
            else if(str1.compareToIgnoreCase(str2) > 0){
                  System.out.println("'"+str1+"' is lexicographically bigger than string '"+str2+"'");
            }
            else{
                  System.out.println("'"+str2+"' is lexicographically bigger than string '"+str1+"'");
            }
      }
      public static void main(String[] args) {
            String str1,str2;
            Scanner sc=new Scanner(System.in);
            System.out.print("Enter First String : ");
            str1=sc.nextLine();
            System.out.print("Enter Second String : ");
            str2=sc.nextLine();
            compare(str1, str2);
            compareLexicograph(str1, str2);
            compareIgnoreCase(str1, str2);
      }
}

File: cognizent_GENCNEXT/CompareString.cs
using System;

namespace cognizent_GENCNEXT
{
    public class CompareString
    {

        public static void compare(string str1, string str2)
        {
            if (str1.Equals(str2))
            {
                Console.WriteLine("Both input string are equal !!");
            }
            else
            {
                Console.WriteLine("Both input string are not equal !!");
            }
        }
        public static void compareLexicograph(string str1, string str2)
        {
            if (str1.CompareTo(str2) == 0)
            {
                Console.WriteLine("Both input string are equal !!");
            }
            else if (str1.CompareTo(str2) > 0)
            {
                Console.WriteLine("'" + str1 + "' is lexicographically bigger than string '" + str2 + "'");
            }
            else
            {
                Console.WriteLine("'" + str2 + "' is lexicographically bigger than string '" + str1 + "'");
            }
        }
        public static void compareIgnoreCase(string str1, string str2)
        {
            if (string.Compare(str1, str2, StringComparison.OrdinalIgnoreCase) == 0)
            {
                Console.WriteLine("Both input string are equal !!");
            }
            else if (string.Compare(str1, str2, StringComparison.OrdinalIgnoreCase) > 0)
            {
                Console.WriteLine("'" + str1 + "' is lexicographically bigger than string '" + str2 + "'");
            }
            else
            {
                Console.WriteLine("'" + str2 + "' is lexicographically bigger than string '" + str1 + "'");
            }
        }
        public static void Main(string[] args)
        {
            string str1, str2;
            Console.Write("Enter First String : ");
            str1 = Console.ReadLine();
            Console.Write("Enter Second String : ");
            str2 = Console.ReadLine();
            compare(str1, str2);
            compareLexicograph(str1, str2);
            compareIgnoreCase(str1, str2);
        }
    }
}
